Tests/FilesTests/FilesTests.swift

```diff
@@ -698,6 +698,26 @@ class FilesTests: XCTestCase {
             try assert(subfolder.file(named: "file"), throwsError: File.PathError.invalid(file.path))
         }
     }
+
+    func testFolderContainsFile() {
+        performTest {
+            let fileA = try FileSystem().temporaryFolder.createFile(named: UUID().uuidString)
```

Comment:
I think it's better if we - as much as possible - create all of our testing files within the contained `folder`. That way we can more easily clean up after ourselves. How about creating a subfolder in `folder` and using that for verification instead?

Reply:
👍